Packs/NetskopeV2/Integrations/NetskopeV2/NetskopeV2.py

```diff
@@ -234,7 +234,9 @@ def main() -> None:
     proxy = demisto.params().get('proxy', False)
 
     headers = {
-        'Netskope-Api-Token': api_key
+        'Content-Type': 'application/json',
+        'Netskope-Api-Token': api_key,
+        'Accept': 'application/json'
```

Comment:
did you test it that it worked with the new headers? if so please attach a screenshot showing that it worked after fix

Reply:
hey @sorkan 
please address this comment.